rules/integrations/aws/privilege_escalation_aws_suspicious_saml_activity.toml

```diff
@@ -0,0 +1,71 @@
+[metadata]
+creation_date = "2021/09/22"
+maturity = "production"
+updated_date = "2021/09/22"
+integration = "aws"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when suspicious SAML activity has occurred in AWS. An adversary could gain backdoor access via SAML manipulation.
```

Comment:
If we don't differentiate between legit activity, we should remove the "suspicious" here